README.md

```diff
@@ -1826,27 +1828,32 @@ _You can enable the following settings in Xcode by running [this script](resourc
     case neptune = 7
   }
 
+  // WRONG
   enum ErrorCode: Int {
     case notEnoughMemory
     case invalidResource
     case timeOut
   }
 
   // RIGHT
+  // Relying on Swift's automatic enum values
   enum ErrorType: String {
     case error
     case warning
   }
 
-  /// These are written to a logging service. Explicit values ensure they're consistent across binaries.
+  // RIGHT
   // swiftformat:disable redundantRawValues
+  /// These are written to a logging service. Explicit values ensure they're consistent across binaries.
   enum UserType: String {
     case owner = "owner"
     case manager = "manager"
     case member = "member"
   }
-  // swiftformat:enable redundantRawValues
 
+  // RIGHT
+  // swiftformat:enable redundantRawValues
```

Comment:
This should be:

```suggestion
  // swiftformat:disable redundantRawValues
```

with the corresponding `// swiftformat:enable redundantRawValues` directive after the closing `}` of the enum

Reply:
@calda Sorry to bother you again, but I am confused with "redundant" raw values and usage of swiftformat. 

As I have read rules.md of swiftformat and your comment for serveral times, I think 
1) `enum ErrorType: String' in WRONG is the one that needs to be implemented with `swiftformat:disable redundantRawValues` and `swiftformat:enable redundantRawValues` as it needs to be excepted from swiftformat rules.
2) `enum Planet: Int` needs to be reverted as before, becuase it is straightly following Swift's automatic enum values, and it does not contain any redundant raw values.

May I hear back what you think on these two?
Thank you for your time in advance.

Reply:
Oops! Sorry, you're right -- my mistake. Mind making another PR to fix that?

Reply:
Thank you for getting back to me! I am on it. Allow me few seconds please! :)